app/views/comments/update.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1>Comments Update</h1>
+
+<%= form_for @comment do |f| %>
+    <%= f.label :content %>
+    <%= f.text_area :content, class: 'form-control' %>
+    <!-- <%= f.hidden_field :post_id, value: @post.id %> -->
```

Comment:
Leaving commented code in your commit is a bad practice. Please remove.

Reply:
Sure will do that. Thanks for picking that out.